src/components/MewSheet/MewSheet.vue

```diff
@@ -1,31 +1,31 @@
-    <!--
-  =====================================================================================
-    Mew Sheet 
-    TODO: need to come back to this to add dark mode and figure out if we actually 
-    need this sheet
-    slot: content (used to place custom ui in the body)
-  =====================================================================================
-  -->
+<!--
+=====================================================================================
+  Mew Sheet 
+  TODO: need to come back to this to add dark mode and figure out if we actually 
+  need this sheet
+  slot: content (used to place custom ui in the body)
+=====================================================================================
+-->
 <template>
   <v-sheet
-    min-width="600"
-    width="100%"
-    color="white"
-    class="mew-sheet rounded-lg"
```

Comment:
can you add back the mew-sheet class to be used for styling the dark mode box shadow

Reply:
Fixed